### DISCUSSION TITLE
/records page has values but no dates

### DISCUSSION THREAD

**Discussion by pbenven** [poblabs/weewx-belchertown#952] [Q&A]
Some of the data points on the /records page have values but do not have the relevant timestamp of when they occurred. It seems random - "Rain" has some, "Humidity" and "Barometer" are missing completely. Recently upgraded weewx to 5.1 from 4.10 - running as root with a virtualized environment but pointing to original /home/weewx/ data and config.

http://littledoorfarm.com/wx/records

**Comment by pbenven** [comment]
Never mind - reverted the whole shebang back to 4.10. I don't have time for this right now.

**Comment by gnartS** [comment]
Sadly not an answer but i have the same issue - dates not showing on records for anything except temperatures and wind gusts.
Upgraded from v4 to v5.

**Comment by uajqq** [comment]
This repo is no longer being actively maintained, so it might be an old bug. I've been trying to keep up with bug fixes on my own fork, [https://github.com/uajqq/weewx-belchertown-new](https://github.com/uajqq/weewx-belchertown-new). Can you install that and see if the problem persists?
